### PR TITLE
Switch polarity for failing on mapping errors

### PR DESCRIPTION
**Pull request by viveklak**

### CODE REVIEW

File: provider-ci/src/workflows.ts [hunk 1]
```diff
@@ -264,8 +264,8 @@ export function UpdatePulumiTerraformBridgeWorkflow(
       // If there are missing or extra mappings, they can not have been
       // introduced by updating the bridge, so for this workflow we'll
       // ignore mapping errors.
-      PULUMI_EXTRA_MAPPING_ERROR: false,
-      PULUMI_MISSING_MAPPING_ERROR: false,
+      PULUMI_SKIP_EXTRA_MAPPING_ERROR: true,
```
Comment (viveklak) [inline provider-ci/src/workflows.ts:267]: Not convinced this is what we want to do here but sticking with the existing semantics for now in the case of bridge upgrades.

**Comment by jkodroff** [comment]
I believe AliCloud will need an exception to missing mappings.

**Comment by viveklak** [comment]
> I believe AliCloud will need an exception to missing mappings.

That is done here: https://github.com/pulumi/ci-mgmt/pull/269/files#diff-ebf701c14f061a2f57de60aa77b9b6e45cc4c1cbc3f22dacd756f58d728b9ad1R15

**Review by lblackstone — APPROVED** [review]

**Comment by viveklak** [comment]
I won't merge this till we fix the builds for TF provider.

**Pull request closed without merging**